fmriprep/cli/parser.py

```diff
@@ -99,6 +99,11 @@ def _bids_filter(value):
              "https://fmriprep.readthedocs.io/en/%s/faq.html#"
              "how-do-I-select-only-certain-files-to-be-input-to-fMRIPrep" % (
                  currentv.base_version if is_release else 'latest'))
+    g_bids.add_argument(
+        "--anat-derivatives", action='store', nargs="?", metavar="PATH", type=PathExists,
```

Comment:
why is this setting nargs? can you pass `--anat-derivatives` without any path?

Reply:
Yes, that would point it to the default folder (`<fmriprep_output>/fmriprep/`). But I will double-check just in case.

Reply:
Okay, I have removed the `nargs` because there wasn't any reason to have it. Good catch!